README.md

```diff
@@ -223,3 +223,39 @@ For more information refer to [Using Docker and Docker-Compose][], this page als
 [definitelytyped]: https://definitelytyped.org/
 [puppeteer]: https://developers.google.com/web/tools/puppeteer
 [cdp]: https://chromedevtools.github.io/devtools-protocol/
+
+## Testing Slack Integration
+
+1. Ensure you can access the MSK secret server
+2. Ensure you can access the [Slack API Manager](https://api.slack.com/apps/AQC1QUJQK/interactive-messages)
+3. Install [ngrok](https://ngrok.com/download)
+4. Update the `/src/main/resources/config/application-dev.yml` file
+
+   - Change `spring.mail.username` to dev.oncokb@gmail.com
+   - Change `spring.mail.password` to the password for dev.oncokb@gmail.com found in the secret server
+   - Check your `application.api-proxy-url` points to a running instance of OncoKB core
```

Comment:
Can probably exclude this since we don't need the core to test slack?

Reply:
Agree